setup.py

```diff
@@ -82,7 +82,7 @@ def read_requirements(filename):
                     raise ValueError('Unknown requirement: {0}'
                                      .format(line))
 
-                DEPENDENCY_LINKS.append(repo_link)
+                DEPENDENCY_LINKS.append(line)
```

Comment:
Ah, so every time the moban templates are updated, all repositories should be synced?